examples/tutorials/colabs/Habitat_Interactive_Tasks.ipynb

```diff
@@ -0,0 +1,2228 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Furniture Rearrangement - How to setup a new interaction task in Habitat Lab\n",
+    "\n",
+    "This tutorial demonstrates how to setup a new task in Habitat that utilizes interaction capabilities in Habitat Simulator.\n",
+    "\n",
+    "![teaser](https://drive.google.com/uc?id=1pupGvb4dGefd0T_23GpeDkkcIocDHSL_)\n",
+    "\n",
+    "## Task Definition:\n",
+    "The working example in this demo will be the task of **Furniture Rearrangement** - The agent will be randomly spawned in an environment in which the furniture are initially displaced from their desired position. The agent is tasked with navigating the environment, picking furniture and putting them in the desired position. To keep the tutorial simple and easy to follow, we will rearrange just a single object.\n",
+    "\n",
+    "To setup this task, we will build on top of existing API in Habitat-Simulator and Habitat-Lab. Here is a summary of all the steps involved in setting up this task:\n",
+    "\n",
+    "1. **Setup the Simulator**: Using existing functionalities of the Habitat-Sim, we can add or remove objects from the scene. We will use these methods to spawn the agent and the objects at some pre-defined initial configuration.\n",
+    "2. **Create a New Dataset**: We will define a new dataset class to save / load a list of episodes for the agent to train and evaluate on.\n",
+    "3. **Grab / Release Action**: We will add the \"grab/release\" action to the agent's action space to allow the agent to pickup / drop an object under a crosshair.\n",
+    "4. **Extend the Simulator Class**: We will extend the Simulator Class to add support for new actions implemented in previous step and add other additional utility functions\n",
+    "5. **Create a New Task**: Create a new task definition, implement new *sensors* and *metrics*.\n",
+    "6. **Train an RL agent**: We will define rewards for this task and utilize it to train an RL agent using the PPO algorithm.\n",
+    "\n",
+    "Let's get started!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# @title Installation { display-mode: \"form\" }\n",
+    "# @markdown (double click to show code).\n",
+    "\n",
+    "!curl -L https://raw.githubusercontent.com/facebookresearch/habitat-sim/master/examples/colab_utils/colab_install.sh | NIGHTLY=true bash -s\n",
+    "%cd /content\n",
+    "\n",
+    "!gdown --id 1Pc-J6pZzXEd8RSeLM94t3iwO8q_RQ853\n",
+    "!unzip /content/coda.zip -d /content/habitat-sim/data/scene_datasets"
```

Comment:
You need to have a flag to allow overwriting or skipping the previous unzip, otherwise it pauses execution and waits for user input